rachellegelden/Home-Together-Canada
# HTC- 370: Sending Request for Member Search Results from FE

# [HTC-370](https://github.com/rachellegelden/Home-Together-Canada/issues/370)
Closes #370 

## Summary
- send request from client to server with search filters and location, and receives back a list of profiles match the search
- added areas of interest and roommates to profile object that is returned by the member search

## Relevant Motivation & Context
This step is required to get the necessary information that will be displayed in the search results

## Testing Instructions
- import the following collection into postman - https://www.getpostman.com/collections/e9f690da4566ba511c37
    - see [here](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#:~:text=To%20import%20your%20API%20specifications,bring%20your%20data%20into%20Postman.) for how to do this
    - be sure to stagger the requests by setting a delay time
![image](https://user-images.githubusercontent.com/43148498/108034742-d2869d80-6fea-11eb-8777-4dbf937b8439.png)
- login to a user on the client, and select some filters (that will return users), and then click search. You should get results in the console (I'll take this console log out in the next ticket)

## Notes
- I think there is a console error  that has to do with the radius type. I'll fix this in the next ticket


## Developer checklist prior to opening this pull request:


## Reviewer
